<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QMainWindow" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editPwd">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>密码：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonConfirm">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车辆信息管理系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
